internal/component/postgresql/postgresdbaccess.go

```diff
@@ -216,13 +216,12 @@ func (p *PostgresDBAccess) doSet(parentCtx context.Context, db dbquerier, req *s
 
 	result, err := db.Exec(parentCtx, query, params...)
 	if err != nil {
-		if req.ETag != nil && *req.ETag != "" {
-			return state.NewETagError(state.ETagMismatch, err)
-		}
 		return err
 	}
-
 	if result.RowsAffected() != 1 {
```

Comment:
This code matches another place in this file that specifically checks `rows != 1`. We could make it `rows == 0` but it doesn't really matter since `rows > 1` isn't possible in these cases.